<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DvdDialog</class>
 <widget class="QDialog" name="DvdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Dvd</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>14</number>
     </property>
     <property name="verticalSpacing">
      <number>14</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dvd Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="dvdTitle"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Director</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dvdDirector"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="dvdYearSpinbox">
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>2100</number>
       </property>
       <property name="value">
        <number>2020</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Quantity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="dvdQuantitySpinbox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="dvdLengthSpinbox">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="dvdDescriptionTextEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Genres</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QListWidget" name="dvdGenreList">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Action</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Anime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Comedy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Drama</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fantasy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Horror</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mystery</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Romance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thriller</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Western</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="acceptButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>DvdDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>DvdDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
